packages/rocketchat-ui-sidenav/client/roomList.js

```diff
@@ -73,6 +73,15 @@ Template.roomList.helpers({
 	}
 });
 
+Template.roomList.rendered = function() {
+	if (this.data.identifier !== undefined && this.data.identifier === 'c') {
+		const _first_child = this.find('.channelItemLists').children[0];
+		if (_first_child !== '' && _first_child !== undefined) {
+			_first_child.children[0].click();
```

Comment:
Please look at room manager to open channels.  The DOM elements here are going to change.  If this class goes away the functionality breaks.